nats-server/configuration/gateways/README.md

```diff
@@ -70,7 +77,7 @@ When a gateway on _A_ sends many messages on various subjects for which _B_ has
 
 When a queue subscriber creates a new subscription, the gateway propagates the subscription interest to other gateways. The subscription interest is only propagated _once_ per _Account_ and subject. When the last queue subscriber is gone, the cluster interest is removed.
 
-Queue subscriptions work on _Interest-only Mode_ to honor NATS' queue semantics across the _Super Cluster_. For each queue group, a message is only delivered to a single queue subscriber. Only when a local queue group member is not found, is a message forwarded to a different interested cluster; gateways will always try to serve local queue subscribers first and only failover when a local queue subscriber is not found.
+Queue subscriptions work on _Interest-only Mode_ to honor NATS' queue semantics across the _Super Cluster_. For each queue group, a message is only delivered to a single queue subscriber. If the same queue group exists in multiple clusters, the server will pick one member from the queue group in its cluster, only sending to a different cluster if there is no interest in its cluster. In other words, a server will always try to serve local queue subscribers first and only failover when a local queue subscriber is not found. The server will pick the cluster with the lowest RTT.
```

Comment:
>  only sending to a different cluster if there is no interest in its cluster.

After re-reading it, `in its cluster` seem redundant. So simply `if there is no local interest`. It's just that I hope user do not interpret "local" with server-only. In the context of gateways and subscription interest, here "local" means all nodes in a given cluster.